benchmark.py

```diff
@@ -332,6 +332,32 @@ def _run_service_test(example, service, test_suite_config, namespace):
         else:
             print(f"[OPEA BENCHMARK] 🚀 Dataset is not specified for {service_name}. Check the benchmark.yaml again.")
 
```

Comment:
Does DocSum required an ingest data process, if yes, need to update this part as well.